Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1390,34 +1392,38 @@ void MenuBar::SaveSymbolMap()
 
 void MenuBar::LoadOtherSymbolMap()
 {
-  const QString file = QFileDialog::getOpenFileName(
-      this, tr("Load map file"), QString::fromStdString(File::GetUserPath(D_MAPS_IDX)),
-      tr("Dolphin Map File (*.map)"));
+  Core::RunAsCPUThread([this] {
```

Comment:
I think bad things will happen if you call Qt code from a different thread. In general, threads that aren't the UI thread are not allowed to do any kind of UI operation.

Reply:
`RunAsCPUThread` (not to be confused with `RunOnCPUThread`) temporarily turns the current thread into the CPU thread and then runs the code in the current thread. So the code will still be running from the UI thread.

Reply:
I can still take it out of the lambda.  I think I would prefer it that way anyhow.